tests/swarm-test.js

```diff
@@ -25,164 +28,89 @@ process.on('uncaughtException', function (err) {
 })
 
 experiment('Without a Stream Muxer', function () {
-  experiment('tcp', function () {
+  experiment('and one swarm over tcp', function () {
```

Comment:
Do we need this degree of separation between 'one swarm' and 'two swarms'. 'one swarm' is just to test if the construction doesn't throw.